modules/local/bowtie_map_mirna.nf

```diff
@@ -24,16 +22,14 @@ process BOWTIE_MAP_SEQ {
     path index
 
     output:
-    tuple val(meta), path("*bam"), emit: bam
-    tuple val(meta), path('unmapped/*fq.gz') , emit: unmapped
-    path "*.version.txt" , emit: versions
+    tuple val(meta), path("*bam")           , emit: bam
+    tuple val(meta), path('unmapped/*fq.gz'), emit: unmapped
+    path "versions.yml"                     , emit: versions
 
     script:
-    def software = getSoftwareName(task.process)
-    def process_name = task.process.tokenize(':')[-1]
+    // def process_name = task.process.tokenize(':')[-1] //TODO check if is still needed
```

Comment:
same here

Reply:
I updated the publishDir of all the bowtie local modules here: a5ae9fc